compiler/tools/fpp-to-cpp/test/component/impl/ActiveCommands.ref.template.cpp

```diff
@@ -0,0 +1,380 @@
+// ======================================================================
+// \title  ActiveCommands.template.cpp
+// \author Generated by fpp-to-cpp
+// \brief  cpp file for ActiveCommands component implementation class
+// ======================================================================
+
+#include "FpConfig.hpp"
+#include "impl/ActiveCommands.hpp"
+
+// ----------------------------------------------------------------------
+// Component construction, initialization, and destruction
+// ----------------------------------------------------------------------
+
+ActiveCommands ::
+  ActiveCommands(const char* const compName) :
+    ActiveCommandsComponentBase(compName)
+{
+
+}
+
+void ActiveCommands ::
```

Comment:
@LeStarch Did you want us to remove this init function from the code gen? I seem to recall a change in F Prime along these lines.

Reply:
Yes please!  The we now recommend that projects use the existing `init` function as it is all boilerplate.  Thus we've been removing them from new components by hand.

Reply:
Thanks! This is done.